packages/edit-site/src/plugins/index.js

```diff
@@ -16,6 +22,29 @@ registerPlugin( 'edit-site', {
 		return (
 			<>
 				<ToolsMoreMenuGroup>
+					<MenuItem
+						role="menuitem"
+						icon="download"
+						onClick={ () =>
+							apiFetch( {
+								path: '/edit-site/v1/export',
+								parse: false,
+							} )
+								.then( ( res ) => res.blob() )
+								.then( ( blob ) =>
+									downloadjs(
```

Comment:
Is there not a built-in way to download a file from WordPress? I don't like that we have to add a whole new dependency for it 😕

Reply:
Not that I know of. It's a tiny browser-compat shim.

Reply:
Can't we just "link" to the download URL instead of using apiFetch?

Reply:
We would need to persist the file in a public directory. This seemed cleaner.

Reply:
I'm pretty sure that's not needed since the request already sends a zip file but maybe there's something I'm missing.

Reply:
Oh, you mean a link to the URL using an `href`. I thought you meant having the API return the link to the file.

That won't work either, because we need the nonce middleware and all that for authentication. Unless you think there is another way to authenticate there?

Reply:
I believe if you use a link, you might not need the nonces and things like that as it's a "GET" request and it uses the session right?

Reply:
I tried it, but it's still needed.

The API fetch is also a "GET" request.

Reply:
It's definitely weird to bring in `downloadjs` just for this, even if it's not a huge lib.

The limitation with directly accessing the endpoint lies not in the nonce requirement, since that can be [satisfied with the `_wpnonce` GET parameter](https://developer.wordpress.org/rest-api/using-the-rest-api/authentication/#cookie-authentication), but in the authentication itself.

My reflex here was that we should try to make this work with plain URLs, but after digging a bit I don't think it's very practical.

However, depending on `downloadjs` is a little unfortunate, and looking at its [source](https://github.com/rndme/download/blob/master/download.js) it seems like it does a lot of legwork to support older browsers that we don't need to care about anymore, not to mention that it supports different download methods (e.g. URL-based). Could we replace it with something dead-simple? The following works for me in my browser console:

```js
function downloadBlob( data, filename, mimeType ) {
  const blob = new Blob( [ data ], { type: mimeType } );
  const url = URL.createObjectURL( blob );
  const anchor = document.createElement( 'a' );
  anchor.download = filename;
  anchor.href = url;
  document.body.appendChild( anchor );
  anchor.click();
  document.body.removeChild( anchor );
}
```

If this is worthwhile I can open a PR.